zerver/lib/notifications.py

```diff
@@ -327,7 +327,8 @@ def do_send_missedmessage_events_reply_in_zulip(user_profile, missed_messages, m
     email_dict = {
         'template_prefix': 'zerver/emails/missed_message',
         'to_email': display_email(user_profile),
-        'from_email': from_email,
+        'from_name': from_name,
+        'from_address': from_address,
         'reply_to_email': address,
```

Comment:
I wonder if this should be formataddr((from_name, address)), at least when from_address != FromAddress.NOREPLY. (as a separate commit / followup). I'd also rename `address` here to something else, now that we have a `from_address` variable.